tools/pywasmcross

```diff
@@ -104,6 +104,8 @@ def handle_command(line, args):
     for arg in line:
         if r'/file.c' in arg or '_configtest' in arg:
             return
+        if re.match('/tmp/.*/source\.[bco]+', arg):
```

Comment:
Looks good.  Once Circle-CI finishes successfully, I think this is good to merge.